metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -96,13 +98,15 @@ final class BloopServers(
         serverStarted
       )
 
+    val finished = Promise[Unit]()
```

Comment:
This promise is used to determine whether the launcher is finished.

Reply:
then maybe it could be called: `launched` instead of `finished`?

Reply:
`finished` is updated when the launcher finishes and not when it launches, so I think this is better.